inc/Engine/CriticalPath/Admin/Admin.php

```diff
@@ -51,21 +51,22 @@ public function cpcss_heartbeat() {
 			return;
 		}
 
-		$cpcss_pending = (array) get_transient( 'rocket_cpcss_generation_pending' );
+		$cpcss_pending = get_transient( 'rocket_cpcss_generation_pending' );
 
 		if ( ! empty( $cpcss_pending ) ) {
 			$cpcss_pending = $this->process_cpcss_pending_queue( $cpcss_pending );
+		} elseif ( false !== $cpcss_pending ) {
+			delete_transient( 'rocket_cpcss_generation_pending' );
```

Comment:
For notes from Remy and mine conversation in Slack:

If the transient exists but is empty, we need to delete the transient. Why? Currently, it's deleted in `generation_complete()`. But that method has many guard clauses, meaning the transient may not be deleted in that method.

If the transient doesn't exist, `get_transient` returns `false`. If the transient is empty but does exist, here we delete it to clean up after ourselves.